src/resources/views/crud/columns/inc/wrapper_end.blade.php

```diff
@@ -1 +1,12 @@
-</{{$column['wrapper']['element'] ?? 'a'}}>
\ No newline at end of file
+@php
+    // this is made available by columns like select and select_multiple
+    $related_key = $related_key ?? null;
+
+    // define the wrapper element
+    $wrapperElement = $column['wrapper']['element'] ?? 'a';
+    $wrapperElement = !is_string($wrapperElement) && is_callable($wrapperElement)
+        ? $wrapperElement($crud, $column, $entry, $related_key)
+        : $wrapperElement ?? 'a';
```

Comment:
Guys, is it really necessary to run this logic again in `wrapper_end`? Wouldn't `$column['wrapper']` already be properly defined when it gets here, since `wrapper_start` gets run before `wrapper_end`?

Reply:
@tabacitu I think we may need that, inside the `@include('wrapper_start')`, I'm not able to edit the `$column` variable, to save the state for `wrapper_end`. Do you now any way to do that?